src/components/RouteWrapper.tsx

```diff
@@ -296,6 +297,13 @@ export const ROUTES: RouteProperties[] = [
         description: "A network scanning tool used to identify open ports and services on targeted hosts.",
         category: "Network Scanning and Enumeration",
     },
+    {
+        name: "Amass",
+        path: "/tools/Amass",
+        element: <Amass />,
+        description: "Amass is a tool for in-depth DNS enumeration and network mapping",
+        category: "Information Gathering and Reconnaissance",
```

Comment:
The category should be "Information Gathering and Analysis" 
```suggestion
        category: "Information Gathering and Analysis",
```